klone.yaml

```diff
@@ -0,0 +1,59 @@
+# This klone.yaml file describes the Makefile modules and versions that are
+# cloned into the "make/_shared" folder. These modules are dynamically imported
+# by the root Makefile. The "make upgrade-klone" target can be used to pull
+# the latest version from the upstream repositories (using the repo_ref value).
+#
+# More info can be found here: https://github.com/cert-manager/makefile-modules
+
+targets:
+  make/_shared:
+    - folder_name: generate-verify
+      repo_url: https://github.com/cert-manager/makefile-modules.git
+      repo_ref: main
+      repo_hash: 2547c81aaa2ff4aeefdda53988191b5cbe929985
+      repo_path: modules/generate-verify
+    - folder_name: go
+      repo_url: https://github.com/cert-manager/makefile-modules.git
+      repo_ref: main
+      repo_hash: 2547c81aaa2ff4aeefdda53988191b5cbe929985
+      repo_path: modules/go
+    - folder_name: helm
+      repo_url: https://github.com/cert-manager/makefile-modules.git
+      repo_ref: main
+      repo_hash: 2547c81aaa2ff4aeefdda53988191b5cbe929985
+      repo_path: modules/helm
+    - folder_name: help
+      repo_url: https://github.com/cert-manager/makefile-modules.git
+      repo_ref: main
+      repo_hash: 2547c81aaa2ff4aeefdda53988191b5cbe929985
+      repo_path: modules/help
+    - folder_name: kind
+      repo_url: https://github.com/cert-manager/makefile-modules.git
+      repo_ref: main
+      repo_hash: 2547c81aaa2ff4aeefdda53988191b5cbe929985
+      repo_path: modules/kind
+    - folder_name: klone
+      repo_url: https://github.com/cert-manager/makefile-modules.git
+      repo_ref: main
+      repo_hash: 2547c81aaa2ff4aeefdda53988191b5cbe929985
+      repo_path: modules/klone
+    - folder_name: oci-build
+      repo_url: https://github.com/cert-manager/makefile-modules.git
+      repo_ref: main
+      repo_hash: 2547c81aaa2ff4aeefdda53988191b5cbe929985
+      repo_path: modules/oci-build
+    - folder_name: oci-publish
+      repo_url: https://github.com/cert-manager/makefile-modules.git
+      repo_ref: main
+      repo_hash: 2547c81aaa2ff4aeefdda53988191b5cbe929985
+      repo_path: modules/oci-publish
+    - folder_name: repository-base
```

Comment:
I think we should probably skip this module for now. It is very specific for cert-manager repos atm. (eg. creates the OWNERS_ALIASES file).